plugins/woocommerce/tests/e2e-pw/tests/checkout/checkout.spec.js

```diff
@@ -1,672 +1,508 @@
-				).toContainText( '2' );
-				let totalPrice = await page
-					.getByRole( 'row', { name: 'Total' } )
-					.last()
-					.locator( 'td' )
-					.textContent();
-				totalPrice = Number(
-					totalPrice.replace( /\$([\d.]+).*/, '$1' )
-				);
-				await expect( totalPrice ).toBeGreaterThanOrEqual(
-					Number( twoProductPrice )
-				);
-
-				// check the payment methods
-				await expect(
-					page.locator( '#payment_method_bacs' )
-				).toBeEnabled();
-				await expect(
-					page.locator( '#payment_method_cod' )
-				).toBeEnabled();
-			}
+		const loginAtCheckoutState = await updateIfNeeded(
+			api,
+			'settings/account/woocommerce_enable_checkout_login_reminder',
+			'yes'
 		);
 
-		test(
-			'allows customer to fill billing details',
-			{ tag: [ tags.COULD_BE_LOWER_LEVEL_TEST ] },
-			async ( { page } ) => {
-				// this time we're going to add three products to the cart
-				await addOneOrMoreProductToCart( page, simpleProductName, '3' );
-
-				await page.goto( 'checkout/' );
-				await expect(
-					page.locator( 'strong.product-quantity' )
-				).toContainText( '3' );
-				let totalPrice = await page
-					.getByRole( 'row', { name: 'Total' } )
-					.last()
-					.locator( 'td' )
-					.textContent();
-				totalPrice = Number(
-					totalPrice.replace( /\$([\d.]+).*/, '$1' )
-				);
-				await expect( totalPrice ).toBeGreaterThanOrEqual(
-					Number( threeProductPrice )
-				);
-
-				// asserting that you can fill in the billing details
-				await expect(
-					page.locator( '#billing_first_name' )
-				).toBeEditable();
-				await expect(
-					page.locator( '#billing_last_name' )
-				).toBeEditable();
-				await expect(
-					page.locator( '#billing_company' )
-				).toBeEditable();
-				await expect(
-					page.locator( '#billing_country' )
-				).toBeEnabled();
-				await expect(
-					page.locator( '#billing_address_1' )
-				).toBeEditable();
-				await expect(
-					page.locator( '#billing_address_2' )
-				).toBeEditable();
-				await expect( page.locator( '#billing_city' ) ).toBeEditable();
-				await expect( page.locator( '#billing_state' ) ).toBeEnabled();
-				await expect(
-					page.locator( '#billing_postcode' )
-				).toBeEditable();
-				await expect( page.locator( '#billing_phone' ) ).toBeEditable();
-				await expect( page.locator( '#billing_email' ) ).toBeEditable();
-			}
+		const signUpAtCheckoutState = await updateIfNeeded(
+			api,
+			'settings/account/woocommerce_enable_signup_and_login_from_checkout',
+			'yes'
 		);
 
-		test(
-			'warn when customer is missing required details',
-			{ tag: [ tags.COULD_BE_LOWER_LEVEL_TEST ] },
-			async ( { page } ) => {
-				await addAProductToCart( page, productId );
-
-				await page.goto( 'checkout/' );
+		// Check id COD payment is enabled and enable it if it is not
+		const codResponse = await api.get( 'payment_gateways/cod' );
+		const codEnabled = codResponse.enabled;
 
-				// first try submitting the form with no fields complete
```

Comment:
Will these scenarios be added back? Curious why this was considered as COULD_BE_LOWER_LEVEL_TEST.